2016-2017_Starstruck/Swerve Driver Proto/src/SwerveLib.c

```diff
@@ -136,3 +145,243 @@ void crabPID(unsigned char motor, int currentValue, int targetValue, CrabGroup g
 		 );
 	motorSet(motor, -speed);
 }
+
+void pulseMotor(unsigned char x){
+   motorSet(x,127);
+   delay(250);
+   motorSet(x,-127);
+   delay(250);
+   motorSet(x,0);
+ }
+
+void LCD(){
```

Comment:
There are a few in here, wanna keep formatting consistent. If we get a judge that knows code they won't like this